recipe/meta.yaml

```diff
@@ -26,94 +26,267 @@ source:
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - {{ cdt('mesa-libgbm') }}           # [linux]
+        - jom  # [win]
+        - pkg-config  # [not win]
+      host:
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - python
+        - dbus  # [not win]
+        - qt 5.12.*
+        - enum34  # [py27]
+        - {{ pin_subpackage('pyqt-impl', exact=True) }}
+      run:
+        - python
+        - {{ pin_compatible("qt", max_pin="x.x") }}
+        - enum34  # [py27]
+        - {{ pin_subpackage('pyqt-impl', exact=True) }}
+    test:
+      imports:
+        - PyQt5.QtWebEngine
+        - PyQt5.QtWebEngineCore
+        - PyQt5.QtWebEngineWidgets
+
+  - name: pyqtchart
+    version: "5.12"
+    script: build-pyqtcharts.sh   # [not win]
+    script: bld-pyqtcharts.bat  # [win]
+    requirements:
+      run_constrained:                                            # [osx]
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+      build:
+        # We may want to find and add a Python dbus library?
+        # pkg-config --cflags-only-I --libs dbus-1
+        # The Python dbus module doesn't seem to be installed.
+        - {{ compiler('cxx') }}
+        - {{ compiler('c') }}
+        - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+        - {{ cdt('libx11-devel') }}          # [linux]
+        - {{ cdt('libxcb') }}                # [linux]
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('libxrender-devel') }}      # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-libegl-devel') }}     # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('alsa-lib-devel') }}        # [linux]
+        - {{ cdt('gtk2-devel') }}            # [linux]
+        - {{ cdt('gtkmm24-devel') }}         # [linux]
+        - {{ cdt('libdrm-devel') }}          # [linux]
+        - {{ cdt('libxcomposite-devel') }}   # [linux]
+        - {{ cdt('libxcursor-devel') }}      # [linux]
+        - {{ cdt('libxi-devel') }}           # [linux]
+        - {{ cdt('libxrandr-devel') }}       # [linux]
+        - {{ cdt('pciutils-devel') }}        # [linux]
+        - {{ cdt('libxscrnsaver-devel') }}   # [linux]
+        - {{ cdt('libxtst-devel') }}         # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - {{ cdt('mesa-libgbm') }}           # [linux]
+        - jom  # [win]
+        - pkg-config  # [not win]
+      host:
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - python
+        - dbus  # [not win]
+        - qt 5.12.*
```

Comment:
I don't recall why we specify the version here instead of letting the pin do its magic. Do you want to merge as-is or try to remove this? The latter would make it safer for future updates in case it works.

Reply:
I think we merge this as is and then figure this out later. I don't want too many changes in one PR.